_targets/build.common

```diff
@@ -8,6 +8,7 @@
 [ "${BASH_SOURCE[0]}" -ef "$0" ] && echo "You should source this script, not execute it!" && exit 1
 
 KERNEL_ELF="phoenix-${TARGET_FAMILY}-${TARGET_SUBFAMILY}.elf"
+FLASH_IMAGE=n
```

Comment:
uh, failing in `b_add2img` if `FLASH_IMAGE` not set but setting it globally for every target? Please decide on one approach.